tests/unit/dataset/dataset_files_test.py

```diff
@@ -206,7 +206,7 @@ def test_load_eyelink_file(tmp_path, read_kwargs):
         filepath,
         fileinfo_row={},
         definition=DatasetDefinition(
-            experiment=pm.Experiment(1024, 768, 38, 30, None, 'center', 100),
+            experiment=pm.Experiment(1280, 1024, 38, 30, None, 'center', 100),
```

Comment:
why did this change?

Reply:
Same as above. It's different from the resolution logged in the ASC file, but up to now it didn't matter because the mismatch wasn't checked.

Reply:
perfect